smart_contracts.rst

```diff
@@ -84,43 +88,108 @@ Provides the interface to interact with payment channels. The channels can only
 
 .. _channel-identifier:
 
-:term:`Channel Identifier` is currently defined as ``keccak256(address participant1, address participant2)``, where the two participant addresses are in lexicographic order.
+:term:`Channel Identifier` is currently defined as ``uint256``, a global monotonically increasing counter of all the channels inside a ``TokenNetwork``.
```

Comment:
Please mention here that 0 is not a valid value for the identifier of an active channel.

Reply:
I added a note below this definition.